src/split-panel/index.tsx

```diff
@@ -147,6 +147,7 @@ export default function SplitPanel({
       value={{
         stickyOffsetTop: topOffset,
         stickyOffsetBottom: bottomOffset,
+        hasBreadcrumbs: false,
```

Comment:
I'm not sure I understand how this works. Do we set `hasBreadcrumbs` to false because split panel doesn't care about this property?

Reply:
Practically any value does not make sense here, because it is used only by ContentLayout component, which is not expected to be inside split panel.

I put `false` value here as a safer option (no value is better than assuming there is something)